examples/custom-logger.ts

```diff
@@ -0,0 +1,40 @@
+import { SpvWalletClient, generateKeys } from '../dist/typescript-npm-package.cjs.js';
+
+const server = 'http://localhost:3003/v1';
+
+const newXPub = generateKeys().xPub.toString();
+
+// Option 1 - control console log level
+const clientWithCustomLogLevel = new SpvWalletClient(
+  server,
+  {
+    xPub: newXPub,
+  },
+  {
+    //default level is 'info'
+    //the 'console' will be utilized
+    level: 'debug', //available levels: 'debug' | 'info' | 'warn' | 'error' | 'disabled'
+  },
```

Comment:
Check out the `LoggerConfig` type:
```typescript
// ./src/logger.ts
export interface Logger {
  debug(msg: string, ...args: any[]): void;
  info(msg: string, ...args: any[]): void;
  warn(msg: string, ...args: any[]): void;
  error(msg: string, ...args: any[]): void;
}

export interface ConsoleLoggerLevel {
  level: 'debug' | 'info' | 'warn' | 'error' | 'disabled';
}

export type LoggerConfig = Logger | ConsoleLoggerLevel;
```

So... the `level` field is the only one field the user can provide. 

Of course the user can provide the CustomLogger as I'm presenting in this example.

Reply:
Oh, because I saw the comment about `console` and I was wondering if it's configurable as well as `level`. So maybe let's remove the comment about `console` if it's not configurable :)

Reply:
Check it out now 😄